### DISCUSSION TITLE
Suggestion: Emphasize google in the node-red flows site description

### DISCUSSION THREAD

**Discussion by Impact123** [andrei-tatar/node-red-contrib-smartnora#180] [Ideas]
If you search for `google` in [the flows library](https://flows.nodered.org/), Smart NORA won't be found.
I'd suggest editing the description there so the `google` keyword will appear somewhere in the first line and/or add it to the keywords.

**Comment by andrei-tatar** [comment]
Thanks for the suggestion. I updated the description and the keywords, though I think only the description is used for searching.
